apps/openmw/mwmechanics/actors.cpp

```diff
@@ -926,31 +920,27 @@ namespace MWMechanics
         return ctrl->isSneaking();
     }
 
-    void Actors::updateDrowning(const MWWorld::Ptr& ptr, float duration)
+    void Actors::updateDrowning(const MWWorld::Ptr& ptr, float duration, bool isKnockedOut, bool isPlayer)
```

Comment:
Is passing isPlayer that necessary since the boolean is possible to be set in the function itself?

Reply:
I just do not like an idea to get reference to player for every actor.